specification/types.md

```diff
@@ -122,8 +126,19 @@ A structure containing the following fields:
 
 A structure which supports definition of arbitrary properties, with keys of type `string`, and values of type `boolean`, `string`, or `number`.
 
+`key` is a required property with a value of `string`.
+`type` is a required property with a `string` value of either `boolean`, `string`, `number`, or `structure`.
```

Comment:
How should languages that disambiguate number types like Go or Python handle `number`-typed flags?

Reply:
My thought was that this would indicate which client method would be used to resolve the flag. Should we call out that this value simply needs to map to an available client resolution method?